### PR TITLE
Reduced likelihood of ambiguous overload in Property constructors

### PR DESCRIPTION
**Pull request by hernando**

### CODE REVIEW

**Comment by hernando** [comment]
Needs approval again, I hadn't compiled the VRPN plugin.

File: brayns/common/PropertyMap.h [hunk 1]
```diff
@@ -247,6 +247,22 @@ class PropertyMap
     {
     }
 
+    PropertyMap(const PropertyMap& other) = default;
+
+    // std::vector move constructor is not noexcept until C++17, if we want
+    // this class to be movable we have to do it by hand.
+    PropertyMap(PropertyMap&& other) noexcept
```
Comment (tribal-tec) [inline brayns/common/PropertyMap.h]: there is no test for move & copy assignment & construction. This is not needed right now I guess, but you need that for later?

Comment (hernando) [reply brayns/common/PropertyMap.h]: I wanted to have a move constructor and assignment, but copy assignment is already being used. Move construction could also have been used without this, but due to the issue in the comment above, the default move constructor was being deleted (apart from the fact there's a explicit declaration of the copy constructor). As you see, we are not 
Do you really what a test for this? The code is pretty trivial, will that be testing Brayns, the compiler or be a C++11 quiz? (as you can see I'm not an advocate of 100% code coverage)

**Comment by hernando** [comment]
I have another PR on top of this one prepared, do you any other comments?

**Pull request merged**